src/geofenceUtils.ts

```diff
@@ -64,6 +65,36 @@ export const getPolygonFromBounds = (bounds: LngLatBounds): Polygon => {
   return polygon;
 };
 
+export const getCircleFeatureFromCoords = (
+  center: Coordinates,
+  { bounds, radius }: { bounds?: LngLatBounds; radius?: number }
+): Feature<Geometry, GeoJsonProperties> => {
+  if (!bounds && !radius) {
+    throw new Error("Circle requires a bounds or a radius");
+  }
```

Comment:
Is this a user error or a system failure error message? If later, is it recoverable?

Reply:
User error, this is mostly for our own purposes I suppose because this is not a method we will expose to the public

Reply:
Will this ever get bubbled up to the user? If it does then it doesn't really help user to recover from it.

Reply:
Will update this error message, this probably shouldn't get bubbled up to the user but just in case